src/main/kotlin/defineViews.kt

```diff
@@ -8,7 +8,12 @@ import com.structurizr.view.ViewSet
 fun defineViews(model: Model, views: ViewSet) {
   views.createSystemLandscapeView("system-overview", "All systems").apply {
     addAllSoftwareSystems()
-    enableAutomaticLayout(AutomaticLayout.RankDirection.TopBottom, 300, 300)
+
+    // remove 'sign-on' systems to avoid noise
+    remove(HMPPSAuth.system)
+    remove(MoJSignOn.system)
```

Comment:
❤️ this clears up the view substantially. We can always add another view which includes the auth systems if there's value in future, but I guess we'd get everything we need from an auth system diagram. Removing now is a great idea 🙌